yaml/linux/specflow_hyperexecute_autosplit_sample.yaml

```diff
@@ -15,16 +12,13 @@ concurrency: 3
 
 pre:
  - dotnet build
+ - dotnet --version
  
 cacheKey: '{{ checksum "packages.txt" }}'
 
 post:
   - cat yaml/linux/specflow_hyperexecute_autosplit_sample.yaml
 
-runtime:
```

Comment:
it can create confusion with user. can we have same runtime version across all 3 OS ?

Reply:
@abdullah-lt will have to test it then, I will check this pointer in this week when will maintain all the repos. but for now can we please merge this so that if someone triggers it via the quickrun in the mean time then the test will get executed

Reply:
okay please keep this in your bucket